README.md

```diff
@@ -7,80 +7,106 @@
+### Local Development
 
-3. Clone this repository and `cd` into it.
+1. Clone this repository and `cd` into it.
 
    ```bash
    git clone git@github.com:TwilioDevEd/airtng-node.git
    cd airtng-node
    ```
 
-4. Install the dependencies.
+1. Install the dependencies.
 
    ```bash
    npm install
    ```
 
-5. Copy the sample configuration `.env.example` to `.env`, and then edit `.env` to match your configuration.
+1. Copy the sample configuration `.env.example` to `.env`, and then edit `.env` to match your configuration.
 
-  ```bash
-  cp .env.example .env
-  ```
+   ```bash
+   cp .env.example .env
+   ```
 
-  You can find your `TWILIO_ACCOUNT_SID` and `TWILIO_AUTH_TOKEN` in your
-   [Twilio Account Settings](https://www.twilio.com/console).
-   You will also need a `TWILIO_PHONE_NUMBER`, which you may find [here](https://www.twilio.com/console/phone-numbers/incoming).
+   See [Twilio Account Settings](#twilio-account-settings) to locate the necessary environment variables. The phone numbers should be in [E.164 format](https://www.twilio.com/help/faq/phone-numbers/how-do-i-format-phone-numbers-to-work-internationally).
 
-6. Run the application.
+1. Run the application (will run on port 3000).
 
-  ```bash
-  npm start
-  ```
+   ```bash
+   npm start
+   ```
 
-7. Check it out at [http://localhost:3000](http://localhost:3000)
+1. Check it out at [http://localhost:3000](http://localhost:3000)
 
-That's it!
+1. To let our Twilio Phone number use the callback endpoint we exposed, our development server will need to be publicly accessible. You could expose the application to the wider Internet using [ngrok](https://ngrok.com/). [Here](https://www.twilio.com/blog/2015/09/6-awesome-reasons-to-use-ngrok-when-testing-webhooks.html) there is an interesting article about why we recommend you to use ngrok.
+
+   ```bash
+   ngrok http 3000
+   ```
+
+   Keep in mind that our endpoint is:
+
+   ```
+   http://<your-ngrok-subdomain>.ngrok.io/reservations/handle
+   ```
+
+1. You will need to configure Twilio to send requests to your application when SMS are received. You will need to provision at least one Twilio number with sms capabilities so the application's users can make property reservations. On [Twilio Account Settings](#twilio-account-settings) you could find the link to buy a Twilio number. Once you have a number you need to configure your number to work with your application. Open the number management page and open a number's configuration by clicking on it. Remember that the number where you change the SMS webhook must be the same one you set on the TwilioPhoneNumber setting.
+
+   ![Configure Messaging](webhook.png)
 
-To let our Twilio Phone number use the callback endpoint we exposed, our development server will need to be publicly accessible. [We recommend using ngrok to solve this problem](https://www.twilio.com/blog/2015/09/6-awesome-reasons-to-use-ngrok-when-testing-webhooks.html).
+That's it!
 
```

Comment:
Can you add something like this to the readmes?

```suggestion

### Docker

If you have [Docker](https://www.docker.com/) already installed on your machine, you can use our `docker-compose.yml` to setup your project.

1. Make sure you have the project cloned.
2. Setup the `.env` file as outlined in the Local Development steps
3. Run `docker-compose up`
4. Follow the steps in Local Development on how to expose your port to Twilio using a tool like ngrok and configure the remaining parts of your application.


```

Reply:
Hey @dkundel, sure! We will add it to all readme files. Done.